### ISSUE TITLE
MAT-HYPOTHESIS-001: evidence-bounded materials hypothesis proposals and hidden-discovery benchmark

### ISSUE THREAD

**Issue by Tristan-Stoltz-ERC** [Luminous-Dynamics/symthaea#5254]
## Mission

Define the materials-specific boundary between **idea generation** and **scientific evidence**.

Symthaea may use literature reasoning, concept graphs, HDC analogies, LLM/multi-agent reasoning, generative models, causal models, human proposals, or other methods to propose materials hypotheses. None of those proposal mechanisms may self-promote their outputs into scientific evidence, novelty, bottleneck authority, or portfolio value.

The central theorem is:

```text
novel-sounding idea
!= novel hypothesis

novel hypothesis
!= plausible mechanism

plausible mechanism
!= falsifiable hypothesis

falsifiable hypothesis
!= important system bottleneck

important bottleneck
!= successful material intervention

proposal
!= evidence
```

This issue should compose existing Symthaea research/active-inquiry mechanisms rather than create a second general scientific-reasoning engine.

## Current research motivation

Recent 2026 work shows that AI systems can materially assist scientific hypothesis generation and experiment selection:

- Co-Scientist generates and critiques scientific hypotheses for experimental verification: https://www.nature.com/articles/s41586-026-10644-y
- Robin automates hypothesis generation, experiment proposal, data analysis, and hypothesis revision in experimental biology: https://www.nature.com/articles/s41586-026-10652-y
- LLM-based active learning in materials science reported strong sample-efficiency improvements across several materials datasets: https://www.nature.com/articles/s41524-026-02136-4
- concept-graph models can predict emerging research directions in materials science: https://www.nature.com/articles/s42256-026-01206-y
- autonomous-materials-lab strategy work explicitly treats hypothesis generation, validation, rejection, and diversity as research-management variables: https://www.nature.com/articles/s43246-026-01219-5

These are motivation for a rigorous proposal/evaluation boundary, not evidence that any generated materials hypothesis is true or useful.

## Ownership boundary

Do **not** build another generic hypothesis engine here.

Potential generators remain upstream/advisory, including:

- human scientist;
- EUREKA Concept Genesis / active inquiry where applicable;
- general LLM or multi-agent research system;
- literature/concept graph;
- HDC analogy retrieval;
- generative/inverse-design model;
- causal/mechanistic reasoner;
- random/diversity baseline.

MAT-HYPOTHESIS owns only:

1. exact materials-hypothesis proposal semantics;
2. knowledge/exposure provenance;
3. novelty/prior-art audit semantics;
4. falsifiability and discriminating-observation requirements;
5. competing hypotheses/mechanisms;
6. bridge into MAT-OPPORTUNITY / RQ-043 / MAT-014;
7. prospective evaluation of hypothesis-generation quality.

## `MaterialsHypothesisProposalV1`

Bind at minimum:

- exact proposal ID/version;
- generator identity/profile;
- generator artifact/model/source identity where available;
- generation timestamp as provenance only;
- exact declared knowledge snapshot/cutoff;
- technology/function subject when applicable;
- bottleneck hypothesis when applicable;
- material/architecture/process domain;
- exact proposition;
- expected direction/effect;
- proposed mechanism;
- applicability/scope conditions;
- required assumptions;
- competing mechanisms/hypotheses;
- explicit falsifiers;
- observations that would discriminate the proposal from alternatives;
- known supporting evidence refs;
- known contrary evidence refs;
- prior-art/novelty search refs;
- required next evidence classes;
- limitations/unknowns;
- deterministic commitment.

A proposal may be validly represented even when novelty, plausibility, or importance are unknown.

## Proposition structure

Prefer hypotheses that make an externally checkable statement rather than a vague research theme.

Weak:

```text
maybe Fe-Co-X materials are interesting
```

Stronger:

```text
under exact composition/phase/process/temperature scope,
intervention X is predicted to change property Y in direction D
because mechanism M,
while alternative mechanism A predicts outcome A'
```

For system-level hypotheses, bind the exact MAT-OPPORTUNITY-003 system-performance question and bottleneck variable.

## Separate hypothesis classes

Do not force all research ideas into one grammar. At minimum distinguish:

```text
CompositionHypothesis
StructureOrPhaseHypothesis
DefectOrDopingHypothesis
InterfaceHypothesis
MicrostructureHypothesis
ProcessSynthesisHypothesis
MechanismHypothesis
ScalingLawHypothesis
SystemBottleneckHypothesis
SubstitutionHypothesis
RecoverySeparationHypothesis
MeasurementArtifactHypothesis
ModelDiscrepancyHypothesis
```

The evidence needed to falsify each class differs.

## Novelty is not truth

Represent novelty separately from physical plausibility and evidence support.

Suggested states:

```text
NoveltyUnknown
KnownPriorArt
NearbyPriorArt
CombinationAppearsUnreportedUnderSnapshot
MechanismAppearsUnreportedUnderSnapshot
ConflictingNoveltyEvidence
```

Never expose `Novel=true` merely because lexical search found no exact phrase.

Novelty audit should consider where feasible:

- synonyms / nomenclature variants;
- composition aliases;
- structure/prototype aliases;
- equivalent physical mechanisms expressed differently;
- patents as a separate source class;
- preprints vs peer-reviewed literature;
- publication date vs online-first date;
- inaccessible/unindexed literature as uncertainty.

`not found != does not exist`.

## Knowledge / exposure provenance

A hypothesis-generation benchmark is invalid if the generator may already know the held-out discovery.

Bind an `ExposureProfileV1` containing where available:

- literature snapshot cutoff;
- exact corpus/source set;
- model training cutoff declared by provider;
- fine-tuning/RAG corpus identity;
- tool/web access policy during generation;
- memory/context carried into the trial;
- whether exposure to post-cutoff information can actually be ruled out.

Possible classifications:

```text
ExposureControlled
ExposureBoundedByExactCorpus
ProviderDeclaredTrainingCutoffOnly
PostCutoffExposurePossible
ExposureUnknown
```

Do not call a historical rediscovery prospective when post-cutoff exposure cannot be excluded.

## Temporal hidden-discovery benchmark

Build a retrospective benchmark that asks:

> Given only evidence available up to cutoff T, can the generator produce useful, falsifiable hypotheses related to findings that were independently established after T?

But preserve:

```text
later publication
!= ground-truth proof
```

The post-cutoff literature is evidence that the research direction/finding later existed, not automatic truth.

### Strong benchmark requirements

- exact pre-cutoff corpus snapshot;
- exact post-cutoff evaluation corpus held inaccessible to controlled generators;
- candidate generation frozen before evaluation corpus reveal;
- prior-art audit using pre-cutoff material only for novelty-at-T;
- post-cutoff scoring performed independently;
- no tuning on held-out cases before final scoring;
- exact generator/model/tool exposure state.

### Exposure limitation

Modern frontier LLMs may have unknown or post-cutoff pretraining exposure. Therefore use several evidence classes:

1. **controlled-corpus generator** — strongest historical hidden-discovery evidence;
2. **provider-cutoff-bounded generator** — weaker if the provider cutoff is only declared;
3. **open-world contemporary model** — engineering demonstration only, not hidden-discovery evidence;
4. **synthetic hidden-mechanism tasks** — useful construct-validity controls with exact leakage control.

Do not pretend prompt-level date instructions remove pretrained knowledge.

## Evaluation dimensions

Do not use one `hypothesis_quality_score`.

Score independently where possible:

- proposition specificity;
- falsifiability;
- mechanistic coherence;
- consistency with pre-cutoff evidence;
- contradiction awareness;
- novelty under exact snapshot;
- discriminatory power vs alternatives;
- expected observability;
- experimental/computational accessibility;
- system relevance / bottleneck connection;
- later support / contradiction under held-out evidence;
- calibration of stated uncertainty;
- diversity without semantic duplication;
- resource cost to test.

A highly novel but unfalsifiable idea should remain distinct from a mundane but decisive hypothesis.

## Baseline ladder

The first benchmark should compare sophisticated generators against simple controls such as:

```text
B0 random valid combination
B1 nearest-neighbor / adjacent concept recombination
B2 citation-frequency trend extrapolation
B3 concept-graph link prediction
B4 simple retrieval + templated mechanism proposal
B5 one LLM generator
B6 multi-agent / critique-refine generator
B7 Symthaea compositional generator when available
```

Do not assume the most elaborate generator wins.

## Competing-hypothesis requirement

A proposal is stronger when it identifies what else could explain the same observations.

Conceptually:

```text
Hypothesis H1
vs H2/H3 alternatives
-> observation/experiment E
-> outcomes partition hypotheses differently
```

This should bridge to generic RQ-043 information-value machinery for experiment choice.

A proposal with no plausible falsifier or no differentiating observation is `EvidenceDiscriminationWeak`, not automatically rejected as meaningless; some frontier questions may need new measurement infrastructure.

## Bridge to MAT-OPPORTUNITY

MAT-HYPOTHESIS proposals may create or refine candidate:

- `MaterialBottleneckHypothesisV1`;
- intervention candidates;
- mechanism hypotheses;
- opportunity questions.

But:

```text
hypothesis generator says high impact
!= MAT-OPPORTUNITY system leverage
```

MAT-OPPORTUNITY-003 must independently establish/bound system sensitivity and bottleneck attribution where system leverage is claimed.

## Bridge to MAT-016 advisory/evidence firewall

Every generated hypothesis remains advisory/proposal-plane.

There must be no conversion such as:

```text
MaterialsHypothesisProposalV1 -> MAT-008 ConditionedPropertyEvidence
```

without fresh evidence from an independently qualified evaluator/experiment path.

Even a hypothesis that later proves numerically correct remains a proposal in its historical record; later evidence references it rather than rewriting its original authority class.

## Bridge to MAT-014 active discovery

MAT-014 should consume hypothesis proposals as one source of candidate evaluation actions.

Recommended flow:

```text
hypothesis proposal
-> validation / prior-art / bottleneck checks
-> candidate discriminating evidence actions
-> RQ-043 predicted information value
-> MAT-PORTFOLIO resource allocation
-> human/scientific authority
-> execution
-> fresh evidence
-> hypothesis support/contradiction/update
```

MAT-014 should not own a second generic VOI engine or portfolio scheduler.

## Hypothesis revision lineage

Do not mutate one hypothesis object as evidence changes.

Use explicit lineage:

```text
H0 proposal
-> evidence E1
-> H1 refined/split/narrowed successor
```

Preserve whether changes are:

- parameter refinement;
- scope narrowing;
- mechanism revision;
- hypothesis split;
- merger of previously distinct proposals;
- falsification;
- supersession.

A revised hypothesis does not retroactively receive prospective credit for evidence that caused the revision.

## Negative-result memory

Integrate MAT-010 so rejected/falsified hypotheses remain searchable.

Avoid repeatedly regenerating semantically equivalent ideas after they failed under the same scope.

But distinguish:

```text
falsified under profile A
!= falsified universally
```

and allow a new proposal when scope/mechanism/evidence materially changes.

## Generator diversity without fake independence

Multiple agents/models may increase hypothesis diversity, but:

```text
5 agents using same model + same corpus
!= 5 independent scientific sources
```

Track shared generator/model/corpus ancestry.

Agreement among generators is a proposal diagnostic, not physical corroboration.

## First materials benchmark families

Use domains with later-dated discoveries and sufficiently explicit property/mechanism evidence where possible:

- rare-earth-free magnet composition/phase families;
- solid electrolyte transport/stability hypotheses;
- thermoelectric structure/property hypotheses;
- corrosion/coating formulation hypotheses;
- selective-separation material/process hypotheses.

Do not choose only success stories. Include known dead ends, nulls, instability, and misleading correlations.

## Adversarial fixtures

1. hypothesis is lexically novel but physically equivalent to known prior art -> novelty not established;
2. genuinely new combination but violates a known conservation/chemistry constraint -> novelty preserved, plausibility weak;
3. plausible mechanism with no discriminating observation -> falsifiability/discrimination weak;
4. highly testable hypothesis unrelated to system bottleneck -> scientific question valid, system leverage low/unknown;
5. generator saw post-cutoff paper -> historical hidden-discovery claim invalid;
6. prompt says `ignore knowledge after T` but model exposure is unknown -> no controlled-exposure classification;
7. multiple agents repeat the same idea with synonyms -> one semantic proposal family, not diversity inflation;
8. later publication supports candidate but uses incompatible conditions -> no direct validation;
9. later publication contradicts proposal -> preserve negative result;
10. hypothesis revised after evidence -> new lineage, no retroactive prospective credit;
11. LLM confidence high on false proposal -> confidence does not create support;
12. simple concept-graph baseline matches sophisticated generator -> complexity has not earned superiority;
13. generator proposes intervention whose material variable is not system-sensitive -> MAT-OPPORTUNITY-003 blocks leverage claim;
14. hypothesis has excellent expected information value but proposed experiment lacks authority -> no execution;
15. publication/patent search misses an obscure source later found -> novelty state revises, historical search receipt remains unchanged.

### `MAT-HYPOTHESIS-001A` — pure proposal / exposure / falsifier contracts

Execution-free deterministic types only. No generator implementation.

### `MAT-HYPOTHESIS-001B` — prior-art and semantic-equivalence audit contracts

Compose MAT-012/provider provenance; preserve `not found != novel`.

### `MAT-HYPOTHESIS-001C` — controlled temporal hidden-discovery benchmark

Freeze pre/post cutoff corpora and generator exposure profiles before candidate generation.

### `MAT-HYPOTHESIS-001D` — baseline generator ladder

Run simple recombination/concept-graph/retrieval baselines before sophisticated agents.

### `MAT-HYPOTHESIS-001E` — Symthaea/EUREKA proposal bridge

Consume existing Concept Genesis / active-inquiry style outputs as advisory proposals only; do not duplicate their generic reasoning engine.

### `MAT-HYPOTHESIS-001F` — prospective real hypothesis trial

Freeze one genuinely future-facing proposal set before evaluation evidence exists, then score support/contradiction and decision value later.

## Dependencies / relationships

- MAT-016 #4991 advisory/evidence firewall
- MAT-012 #4377 prior art / novelty / metastability / synthesizability separation
- MAT-010 #4341 negative/search memory
- MAT-OPPORTUNITY-001 #5196
- MAT-OPPORTUNITY-002 #5218
- MAT-OPPORTUNITY-003 #5245
- MAT-PORTFOLIO-001 #5247
- MAT-014 #4345 human-authorized active discovery orchestrator
- RQ-043 #2579 generic epistemic action / predicted-vs-realized information value
- EUREKA #2045, especially Concept Genesis / Active Inquiry as generic research-capability work

Do not depend on unqualified #5212 implementation bytes yet; issue-level semantics can proceed independently.

## Claim ceiling

MAT-HYPOTHESIS may establish that a proposal was generated from a declared knowledge state, is specific/falsifiable under a declared grammar, has a bounded prior-art status, and later received support/contradiction under a preregistered evaluation. It does not establish universal novelty, truth, causal mechanism, research importance, commercial value, intellectual-property status, experimental authority, or discovery merely because an AI generated the idea.
